docs/poster_boards.md

```
@@ -0,0 +1,165 @@
+# Poster Boards 関連スキーマ概要
+
+本ドキュメントは、ポスター掲示板（poster_board）に関係する Supabase パブリックスキーマのテーブル構成・リレーションをまとめたものです。随時更新される Enum や RPC も含め、データモデリングの全体像を把握できるようにしています。
+
+## Enum 定義
+
+### poster_board_status
+| 値 | 意味 | 備考 |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)